msk/util.py

```diff
@@ -287,3 +289,57 @@ def get_licenses():
     licenses = glob(join(dirname(__file__), 'licenses', '*.txt'))
     licenses.sort()
     return licenses
+
+
+GIT_IDENTITY_INFO = '''=== Git Identity ===
+msk uses Git to save skills to Github and when submitting a skill to the
+Mycroft Marketplace. To use Git, Git needs to know your Name and
+E-mail address. This is important because every Git commit uses the
+information to show the responsible party for the submission.
+'''
+
+
+GIT_MANUAL_CHANGE_INFO = '''
+Thank you. :)
+
+If you need to change this in the future use
+
+    git --config user.name "My Name"
+
+and
+
+    git --config user.email "me@myhost.com"
+
+'''
+
+
+def ensure_git_user():
+    """Prompt for fullname and email if git config is missing it."""
+    with GitConfigParser(get_config_path('global'), read_only=False) as conf:
+
+        # Make sure a user section exists
+        if 'user' not in conf.sections():
+            conf.add_section('user')
+
+        # Check for missing options and insert them is missing
+        name, email = (None, None)
+        try:
+            name = conf.get('user', 'name')
```

Comment:
I may be reading this incorrectly, but won't this assign the string "name" to the variable `name` if `conf['user']` doesn't exist?

Reply:
Not sure I'm understanding the comment correctly but there may be confusion around the conf object. It's a GitConfigParser object and not a dict. the `get()` method takes a section and an option as the arguments so name is set to the value of name listed in the section user...

I'll see if I can clarify this so it more understandable

Reply:
No fine as it is, was my bad.

Reply:
I've updated the name and use keyword arguments so it's clear the strings refer to section and option.